tests/security/grub_auth/grub_authorization.pm

```diff
@@ -31,7 +31,8 @@ my $test_passwd = 'pw_test';
 sub switch_boot_menu {
     my $switch = shift;
 
-    if (get_var('UEFI')) {
+    # UEFI item is available only from 15-SP4
+    if (get_var('UEFI') && (is_sle('>15-SP3') || is_tumbleweed())) {
```

Comment:
we'd better not using is_tumbleweed 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style
Suggest changes:
  if (get_var('UEFI') && (!is_sle('>15-SP3')))